go/server/templates/search_compare.tmpl

```diff
@@ -0,0 +1,512 @@
+                                            <div class="col-sm-10">
+                                                <input type="text" class="form-control" id="search_total"
+                                                    placeholder=""
+                                                    disabled>
+                                            </div>
+                                        </div>
+                                        <div class="form-group row">
+                                            <label for="reads" class="col-sm-2 col-form-label">reads</label>
+                                            <div class="col-sm-10">
+                                                <input type="text" class="form-control" id="search_reads"
+                                                    placeholder=""
+                                                    disabled>
+                                            </div>
+                                        </div>
+                                        <div class="form-group row">
+                                            <label for="writes" class="col-sm-2 col-form-label">writes</label>
+                                            <div class="col-sm-10">
+                                                <input type="text" class="form-control" id="search_writes"
+                                                    placeholder=""
+                                                    disabled>
+                                            </div>
+                                        </div>
+                                        <div class="form-group row">
+                                            <label for="others" class="col-sm-2 col-form-label">others</label>
+                                            <div class="col-sm-10">
+                                                <input type="text" class="form-control" id="search_others"
+                                                    placeholder=""
+                                                    disabled>
+                                            </div>
+                                        </div>
+                                    </div>
+                                </div>
+                                <div class="form-group row">
+                                    <label for="latency" class="col-sm-2 col-form-label"><span
+                                            style="font-weight:700">latency(Milliseconds)</span></label>
+                                    <div class="col-sm-10">
+                                        <input type="text" class="form-control" id="search_latency"
+                                            placeholder=""
+                                            disabled>
+                                    </div>
+                                </div>
+                                {% endif %}
```

Comment:
nit: can be removed to avoid unnecessary print.

Reply:
Will change that too